### PR TITLE
Creation of Changelog file

### PR DESCRIPTION
**Pull request by nespinoza**
Hi everyone,

 This PR takes on issue #432 of creating a "release notes" document. So far, this only contains the already "done" items for v.1.2, but will update once we have updates on the other items. I'm assigning @hover2pi and @bourque as reviewers for this, as the technical leads.

 Thanks!
Néstor

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Added
+- This `CHANGELOG.rst` file.
+- Citation information in the `README` file (see PR #472).
+- Contribution guide on the project's wiki (https://github.com/ExoCTK/exoctk/wiki/Git-Workflow).
+- Procedures for ExoCTK release (https://github.com/ExoCTK/exoctk/wiki/Release-Procedure).
+- Support for installation via PyPI (see PR #385).
+- JWST instrument throughputs for limb-darkening calculations (see PR #458).
+
+### Removed
+- Dropped support for `python3.6` (see PR #471).
```
Comment (bourque) [inline CHANGELOG.rst:17]:
```suggestion
- Dropped support for Python 3.6 (see PR #471). 
```

Comment (bourque) [reply CHANGELOG.rst:17]: I think this can just be in plain text, unless 'python3.6' is some sort of keyword or filename that I am missing?

**Review by bourque — APPROVED** [review]
Thanks @nespinoza, this looks good!

It looks like this does not really come with any fancy rendering, which is not necessary to have but could make the document a bit more readable.  If you're interested in making it a bit prettier, I would suggest using proper ReStructured Text headings, something like what [`jwql` uses](https://raw.githubusercontent.com/spacetelescope/jwql/develop/CHANGES.rst).  But I think it is also fine to just leave it as it is, so it is your call.

**Review by hover2pi — APPROVED** [review]
Fantastic!

**Pull request merged**